config/300-bitbucketsource.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    eventing.knative.dev/source: "true"
+  name: bitbucketsources.sources.eventing.knative.dev
+spec:
+  group: sources.eventing.knative.dev
+  names:
+    categories:
+    - all
+    - knative
+    - eventing
+    - sources
+    kind: BitBucketSource
```

Comment:
And here `BitBucket` ? less sold on this idea, `bitbuckets` is weird but it might work out.

Reply:
hmm, again, similar comment as above. I think all our sources are using *Source as Kind. 
To maintain consistency, I'd rather leave it as is. But again, if you think we should actually change it, I will do it.

Reply:
cc @grantr